frontend/main/src/main.cpp

```diff
@@ -244,7 +244,7 @@ int main(int argc, char* argv[]) {
         std::string result;
         if (megamol::core::utility::graphics::ScreenShotComments::EndsWithCaseInsensitive(file, ".png")) {
             if (!lua_api.RunString(
-                    megamol::core::utility::graphics::ScreenShotComments::GetProjectFromPNG(file), result)) {
+                    megamol::core::utility::graphics::ScreenShotComments::GetProjectFromPNG(file), result, file)) {
```

Comment:
Also pass file name of screenshot. This is not further used within lua, but sets the lua ScriptPath. The script path is used in GUIWindows to recognise when another project has been loaded via lua. This changed script path is used load the GUI state from the new project file.